### PR TITLE
RD-879 Use new location on windows agent upgrade

### PR DESCRIPTION
**Pull request by geokala**

### CODE REVIEW

File: cloudify_agent/resources/script/windows.ps1.template [hunk 1]
```diff
@@ -54,14 +54,14 @@ function InstallAgent()
     {% if add_ssl_cert %}
     AddSSLCert
     {% endif %}
-    if (-Not (Test-Path "{{ conf.basedir }}\Scripts")) {
+    if (-Not (Test-Path "C:\Program Files\Cloudify {{ version }} Agents\Scripts")) {
```
Comment (YoniItzhak) [inline cloudify_agent/resources/script/windows.ps1.template:57]: Just a general comment. Don't we want to assign "C:\Program Files\Cloudify {{ version }} Agents" to a variable and reuse it in all places?

Comment (geokala) [reply cloudify_agent/resources/script/windows.ps1.template:57]: That seemed to be what conf.basedir was being partially used for; but it's also wrong so not useful for that. I'm checking to see if it crops up anywhere else.

Comment (geokala) [reply cloudify_agent/resources/script/windows.ps1.template:57]: As for refactoring it to have a common variable that's right: Yes, we should probably do that but I'm thinking of holding off on that for the 5.1.1 release unless I find anything to indicate that basedir being wrong will cause flames and sadness.

Comment (geokala) [reply cloudify_agent/resources/script/windows.ps1.template]: OK, I scrapped this bit of code, updated to a different way.

File: cloudify_agent/installer/config/agent_config.py [hunk 2]
```diff
@@ -516,3 +513,10 @@ def update_agent_runtime_properties(cloudify_agent):
         cloudify_agent.pop(item, None)
     ctx.instance.runtime_properties['cloudify_agent'] = cloudify_agent
     ctx.instance.update()
+
+
+def get_windows_basedir():
+    # TODO: Get the program files directory from the machine itself
```
Comment (YoniItzhak) [inline cloudify_agent/installer/config/agent_config.py:519]: A wise developer once told me not to write `TODO` in my code, but rather open a JIRA about it. Which is to say, do we also have a JIRA about it?

Comment (geokala) [reply cloudify_agent/installer/config/agent_config.py:519]: This TODO already existed but yes, we should probably have a jira for it. I'll make a note to add one.

**Review by YoniItzhak — APPROVED** [review]

**Pull request merged**